packages/app-desktop/gui/JoplinCloudLoginScreen.tsx

```
@@ -107,18 +109,21 @@ const JoplinCloudScreenComponent = (props: Props) => {
 			<ButtonBar onCancelClick={() => props.dispatch({ type: 'NAV_BACK' })} />
 		</div>
 	);
 };
 
-const mapStateToProps = (state: AppState) => {
-	return {
-		joplinCloudWebsite: state.settings['sync.10.website'],
-		joplinCloudApi: state.settings['sync.10.path'],
-	};
+const mapStateToProps = (state: AppState, ownProps: { syncTargetId?: number }) => {
+    const syncTargetId = ownProps.syncTargetId ?? 10;
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)